.buildkite/pipeline.yml

```diff
@@ -2,19 +2,39 @@ agents:
   queue: "k8s-builders"
 
 steps:
-  - label: test tls cluster standup
-    key: tls-up
-    command: .buildkite/scripts/standup-tls.sh
+  - label: test tls cluster standup ubuntu
+    key: tls-up-ubuntu
+    command: .buildkite/scripts/standup-tls.sh --prefix=rp-tls-ub --distro=ubuntu-focal
     plugins:
       - docker#v5.4.0:
           image: glrp/atgt:latest
           environment:
             - DA_AWS_ACCESS_KEY_ID
             - DA_AWS_SECRET_ACCESS_KEY
             - AWS_DEFAULT_REGION
-  - label: test tiered-storage-up cluster standup
-    key: tiered-up
-    command: .buildkite/scripts/standup-tiered-storage.sh
+  - label: test tiered-storage-up cluster standup ubuntu
+    key: tiered-up-ubuntu
+    command: .buildkite/scripts/standup-tiered-storage.sh --prefix=rp-tier-ub --distro=ubuntu-focal
+    plugins:
+      - docker#v5.4.0:
+          image: glrp/atgt:latest
+          environment:
+            - DA_AWS_ACCESS_KEY_ID
+            - DA_AWS_SECRET_ACCESS_KEY
+            - AWS_DEFAULT_REGION
+  - label: test tls cluster standup fedora
+    key: tls-up-fedora
+    command: .buildkite/scripts/standup-tls.sh --prefix=rp-tls-fd --distro=Fedora-Cloud-Base-34
+    plugins:
+      - docker#v5.4.0:
+          image: glrp/atgt:latest
+          environment:
+            - DA_AWS_ACCESS_KEY_ID
+            - DA_AWS_SECRET_ACCESS_KEY
+            - AWS_DEFAULT_REGION
+  - label: test tiered-storage-up cluster standup fedora
+    key: tiered-up-fedora
+    command: .buildkite/scripts/standup-tiered-storage.sh --prefix=rp-tier-fd --distro=Fedora-Cloud-Base-34
```

Comment:
Why 34 and not 38? (I just checked https://fedoraproject.org/cloud/download/)

Reply:
Fedora 38 was released like a month ago so my assumption is that people using this will prefer the older, probably better supported version for now. It's just the client tho so it's nbd either way.

Reply:
ok, got it ... but can we use 36 at least :sweat_smile: ? it's what we use in vtools for ducktape ++ snyk complained about fedora 35 (see #999 in vtools)

Reply:
Oh yeah sure, sorry I'm clearly just stuck in the past rn 😂